addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.xml

```diff
@@ -45,7 +45,7 @@
     </t>
 
     <t t-name="mail.DiscussSidebarChannel">
-        <div class="o-mail-DiscussSidebarChannel o-mail-DiscussSidebar-item d-flex align-items-center mx-2 cursor-pointer"
+        <div class="o-mail-DiscussSidebarChannel o-mail-DiscussSidebar-item d-flex rounded-2 align-items-center mx-2 cursor-pointer"
```

Comment:
Ok, though with such roundness on an item we'd expect some spacing between items. Even if it's just 1px